jekyll/_cci2/server-3-overview.adoc

```diff
@@ -14,11 +14,11 @@ toc::[]
 
 == Introduction
 
-CircleCI Server v{serverversion} is a continuous integration and continuous delivery (CI/CD) platform that you can install on your GCP or AWS Kubernetes cluster.
+CircleCI server v{serverversion} is a continuous integration and continuous delivery (CI/CD) platform that you can install on your GCP or AWS Kubernetes cluster.
 
-The core of the CircleCI Server application runs inside Kubernetes.
+The core of the CircleCI server application runs inside Kubernetes.
 
-The application exposes five services using load balancers, three of these load balancers are VPC-internal.
+The application exposes 4 services using load balancers, three of these load balancers are VPC-internal.
```

Comment:
Add a note to say the external load balancer can be made private/internal now with link to more info https://circleci.atlassian.net/browse/SERVER-1156

Reply:
@nathanfishcircleci Just checking, is this the right place for a link to a new "Load Balancers" section, as per the jira ticket mentioned?